docs/modules/ROOT/pages/documentation/microprofile/rest-client.adoc

```diff
@@ -83,3 +83,45 @@ On a per-client basis, or via Microprofile Config properties, the following aspe
 * Keystore for setting up mutual SSL trust
 
 Programmatically it is also possible to provide an implementation of `SslContext` to control all aspects of SSL communication.
+
+[[Automatic-SSL-Context-configuration]]
+== Automatic SSL Context configuration
+
+Now it was enabled the automatic configuration of `SslContext` by following one of next approaches
+
+* Adding the property during the Rest Client call
+
+[cols="1,1", options="header"]
+|===
+|Property
+|Description
+
+|`fish.payara.rest.client.certificate.alias`
+| The alias name of the certificate
+
+|===
+
+==== Example
+
+[source, java]
+----
+RestClientBuilder.newBuilder().baseUri(new URI("http://localhost:8080/service")).property("fish.payara.rest.client.certificate.alias", "someAliasName").build(Service.class);
```

Comment:
you should use https:// in the example since functionality has to do with SSL connections.